extensions/ql-vscode/src/abstract-webview.ts

```diff
@@ -116,6 +117,7 @@ export abstract class AbstractWebview<
       config.view,
       {
         allowInlineStyles: true,
+        allowWasmEval: config.allowWasmEval ?? false,
```

Comment:
Minor, optional: this is slightly more typesafe as it will coerce `allowWasmEval` to a bool.

```suggestion
        allowWasmEval: !!config.allowWasmEval,
```